catalogue/webapp/components/RequestingDayPicker/RequestingDayPicker.tsx

```diff
@@ -1,395 +1,136 @@
-import { FC, useState } from 'react';
+import { FC, forwardRef, MutableRefObject } from 'react';
 import { Moment } from 'moment';
-import DayPickerInput from 'react-day-picker/DayPickerInput';
 import {
   OpeningHoursDay,
   ExceptionalOpeningHoursDay,
 } from '@weco/common/model/opening-hours';
 import { london } from '@weco/common/utils/format-date';
-import LocaleUtils from 'react-day-picker/moment';
-import styled from 'styled-components';
-import AlignFont from '@weco/common/views/components/styled/AlignFont';
-import Icon from '@weco/common/views/components/Icon/Icon';
-import { chevron, calendar } from '@weco/common/icons';
-import { classNames, font } from '@weco/common/utils/classnames';
-import { fontFamilyMixin } from '@weco/common/views/themes/typography';
 import { collectionVenueId } from '@weco/common/services/prismic/hardcoded-id';
 import {
   determineNextAvailableDate,
   convertOpeningHoursDayToDayNumber,
   extendEndDate,
   findClosedDays,
+  isValidDate,
 } from '@weco/catalogue/utils/dates';
 import { usePrismicData } from '@weco/common/server-data/Context';
 import {
   parseCollectionVenues,
   getVenueById,
 } from '@weco/common/services/prismic/opening-times';
+import DateAdapter from '@mui/lab/AdapterMoment';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
```

Comment:
Think one of my comments got lost somehow - I'm a bit worried about the bundle size increase from these, have we looked at the loading in the date picker stuff [dynamically](https://nextjs.org/docs/advanced-features/dynamic-import) only when it's opened?